controllers/common/types.go

```diff
@@ -29,6 +29,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	// AnnotationCleanFinalizer key
+	AnnotationCleanFinalizer = `cleanFinalizer`
```

Comment:
We may need a common prefix for chaos-mesh annotations. @cwen0

Reply:
@yujunz PTAL